src/struct/commands/CommandUtil.js

```diff
@@ -94,71 +94,54 @@ class CommandUtil {
 
     /**
      * Sends a response or edits an old response if available.
-     * @param {string} [content=''] - Content to send.
-     * @param {MessageOptions|MessageAdditions} [options={}] - Options to use.
+     * @param {string|MessageOptions} [options={}] - Options to use.
      * @returns {Promise<Message|Message[]>}
      */
-    async send(content, options) {
-        const transformedOptions = this.constructor.transformOptions(content, options);
-        const hasFiles = (transformedOptions.files && transformedOptions.files.length > 0)
-            || (transformedOptions.embed && transformedOptions.embed.files && transformedOptions.embed.files.length > 0);
+    async send(options) {
+        const hasFiles = options.files && options.files.length > 0;
```

Comment:
Did you check that this change works with files in embeds?

Reply:
As far as I can tell, MessageOptions as of D.js v13 no longer allows files as a property of an embed